eng/tools/rush-runner.js

```diff
@@ -77,7 +77,7 @@ const getServicePackages = (baseDir, serviceDirs) => {
     const packageJsons = getPackageJsons(searchDir);
     for (const filePath of packageJsons) {
       const contents = JSON.parse(fs.readFileSync(filePath, "utf8"));
-      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "mgmt") {
+      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "mgmt" || contents["name"].startsWith("@azure-tests/perf")) {
```

Comment:
I think we should add a new `sdk-type` like `perf-test`, rather than assuming the package name starts with `@azure-tests/perf`.  @HarshaNalluru, what do you think?

Reply:
Or maybe we could just set `sdk-type` to `client` in the perf test projects, but I'm not sure what other side effects this could have.

Reply:
I prefer not to set sdk-type to client because we use this property to identify whether it's data plane(track2 package) or not and run few actions accordingly. For e.g. API review check etc.

sdk-type as "perf-test" is a great option.

Reply:
+1 on "sdk-type": "perf-test"

Reply:
Thanks, @mikeharder,  "perf-test" sounds good.

Reply:
Okay. I will update sdk-type to perf-test for all perf test packages.